src/indicator-fixes.js

```diff
@@ -171,6 +171,18 @@ function normalize245Indicator1(field, record) {
   field.ind1 = field1XX.length === 0 ? '0' : '1';
 }
 
+function noDisplayContantGenerated520Indicator1(field) {
```

Comment:
typo (missing 's') in function name: should be: noDisplayConstantGenerated520Indicator1